manager/orchestrator/updater.go

```diff
@@ -104,7 +104,7 @@ func (u *Updater) Run(ctx context.Context, service *api.Service, tasks []*api.Ta
 	dirtyTasks := []*api.Task{}
 	for _, t := range tasks {
 		if !reflect.DeepEqual(service.Spec.Task, t.Spec) ||
-			!reflect.DeepEqual(service.Endpoint, t.Endpoint) {
+			!reflect.DeepEqual(service.Spec.Endpoint, t.Endpoint.Spec) {
```

Comment:
I had services and tasks that were created with a previous version. The orchestrator did not initialize `Endpoint` in those.